tokio/src/macros/cfg.rs

```diff
@@ -61,6 +61,7 @@ macro_rules! cfg_fs {
     ($($item:item)*) => {
         $(
             #[cfg(feature = "fs")]
+            #[cfg(not(target_arch = "wasm32"))]
             #[cfg_attr(docsrs, doc(cfg(feature = "fs")))]
             $item
```

Comment:
Here you are putting these annotations on lots of stuff in this file. Ideally, I would like to avoid that.

Reply:
This disables filesystem code for Wasm, since Wasm can't access files by itself. I'm not sure how else to remove features. The cfg macros seemed like a nice global way to handle features.

Reply:
the real issue is the asyncify of all fs operations with threads, since wasi does not support threads